website/docs/guides/versioned-resources.markdown

```diff
@@ -59,4 +59,4 @@ terraform import kubernetes_config_map_v1.example default/example
 
 Then run `terraform plan` to confirm that the import was successful. **NOTE: Do not run the plan after renaming the resource in the configuration until after the above steps have been carried out.** 
 
-You can also skip this and just allow Terraform to destroy and recreate the resource, but this is not recommended for resources like `kubernetes_service` and `kubernetes_deployment`. 
\ No newline at end of file
+You can also skip this and just allow Terraform to destroy and recreate the resource, but this is not recommended for resources like `kubernetes_service` and `kubernetes_deployment`. 
```

Comment:
Not sure what exactly has changed here, this is the result of 'make website-lint-fix'.

Reply:
It's just trailing whitespace being remove